tools/chplvis/DataModel.cxx

```diff
@@ -316,23 +336,72 @@ int DataModel::LoadData(const char * filename)
 
       case Ev_task:
         tp = (E_task *)ev;
-        if (++(curTag->locales[curNodeId].numTasks) > curTag->maxTasks)
-          curTag->maxTasks = curTag->locales[curNodeId].numTasks;
-        if (++(tagList[0]->locales[curNodeId].numTasks) > tagList[0]->maxTasks)
-          tagList[0]->maxTasks = tagList[0]->locales[curNodeId].numTasks;
+        //        if (++(curTag->locales[curNodeId].numTasks) > curTag->maxTasks)
+        //          curTag->maxTasks = curTag->locales[curNodeId].numTasks;
+        //        if (++(tagList[0]->locales[curNodeId].numTasks) > tagList[0]->maxTasks)
+        //          tagList[0]->maxTasks = tagList[0]->locales[curNodeId].numTasks;
```

Comment:
Did you really want to keep these lines for some reason (future use), or were you intending to remove them?